specification/columns/negotiatedlistunitprice.md

```diff
@@ -0,0 +1,32 @@
+# Negotiated List Unit Price
+
+The Negotiated List Unit Price represents the agreed-upon unit price for a single [Pricing Unit](#pricingunit) of the associated SKU, inclusive of negotiated discounts, if present, and exclusive of any other discounts. This price is denominated in the [Billing Currency](#billingcurrency). The Negotiated List Unit Price is commonly used for calculating savings based on negotiation activities. If negotiated discounts are not applicable, the Negotiated List Unit Price defaults to the [List Unit Price](#listunitprice).
```

Comment:
>  inclusive of negotiated discounts, if present, and exclusive of any other discounts

Maybe there is a discussion here on the phrasing, and actually the scope.
Here is a way to view things:
- Providers have a public offer, which include discounts for some usages, like commitment-based discounts.
- Customers negotiate with providers, leading to negotiated discounts, which may apply on top of the discounts of the public offer. For example, a customer may negotiated an additional discount on commitment-based discounts.
In that perspective, what is "negotiated discounts, if present, and exclusive of any other discounts" might not be clear on whether it includes or not commitment-based discounts.

As a practitioner, it's important to know the negotiated cost without commitment-based discounts, for 2 use cases:
- Calculate the savings made thanks to the commitments. This relates to the Effective Savings Rate: https://www.finops.org/wg/finops-kpis/?prod_kpis%5Bpage%5D=2#modal-kpi-9650
- Calculate the cost of a workload without the commitments, so as to understand whether effective (amortized) cost variations of the workload are linked to a change in the distribution of the commitment discount or a change in usage.
I am used to call "on-demand cost" the cost of a workload without the commitments. Is it the intent of the negotiated list cost of this PR, or does the negotiated cost intends something different?

Reply:
+1
This also aligns with PricingCategory == "On-Demand" (ad-hoc usage that was not committed).

Reply:
- Please let me know if this version encapsulates what we are looking for and provides more clarity in terms of scope. 
  Any suggestions are welcome :) 
  > The XXX Unit Price represents the agreed-upon unit price for a single Pricing Unit of the associated SKU, inclusive of negotiated discounts, if present, **while exclusive of any commitment-based discounts or any other discounts**...

- Regarding the column names, let's discuss and vote. Please take a look at the [GSpreadsheet](https://docs.google.com/spreadsheets/d/1VlDx7wGH1atBTksQV-sR2ubxbDQjOBdt7cJsQH0SapQ/edit#gid=749340528). As previously mentioned, any feedback, suggestions, or additional criteria would be greatly appreciated.

Reply:
my bad :(

Here is what I should have posted earlier:

> The XXX Unit Price represents the agreed-upon unit price for a single Pricing Unit of the associated SKU, inclusive of negotiated discounts, if present, **while excluding negotiated commitment-based discounts or any other discounts.**

emphasizing the exclusion of the negotiated commitment based discounts and any other...

Reply:
specification/columns/negotiatedlistunitprice.md

Reply:
Maybe we can also make explicit that the the negotiated discount, which is referred to here, is the on the list price:

> "...inclusive of negotiated discounts on the list unit price based on the provider agreement, while excluding other discounts like bundled or commitment-based discounts."

Maybe the sentence starts to be heavy, but it gives more precise contours.